### PR TITLE
[AppConfig]Fix test bug

### PR DESCRIPTION
**Pull request by shenmuxiaosen**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by shenmuxiaosen** [comment]
[Build ](https://dev.azure.com/azure-sdk/public/_build/results?buildId=553002&view=results)timed out after 10 min . Is there a way to retry? @bim-msft

**Review by yungezz — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py [hunk 1]
```diff
@@ -431,7 +431,8 @@ def test_resolve_keyvault(self, key_vault, resource_group):
                  checks=[self.check('[0].key', secret_name),
                          self.check('[0].value', secret_value)])
 
-        exported_file_path = 'export_keyvault.json'
+        exported_file_path = os.path.join(TEST_DIR, 'export_keyvault.json')
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py:434]: hi @shenmuxiaosen is this file `export_keyvault.sjon` deleted by intention?

Comment (bim-msft) [reply src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/test_appconfig_commands.py:434]: @yungezz Yes, we let them make this change. The deleted file was incorrectly located in the root folder of the project.

**Review by bim-msft — APPROVED** [review]

**Pull request merged**